homeassistant/components/meater/sensor.py

```diff
@@ -146,13 +146,13 @@ def async_update_data():
         if not coordinator.last_update_success:
             return
 
-        devices = coordinator.data
+        devices: dict[str, MeaterProbe] = coordinator.data
```

Comment:
We wouldn't need the type info here if we would type the generic coordinator variable completely above.

Reply:
Fixed in https://github.com/home-assistant/core/pull/71333